app/assets/javascripts/components/middleware-generic-properties.js

```diff
@@ -0,0 +1,58 @@
+ManageIQ.angular.app.component('mwGenericPropertiesComponent', {
```

Comment:
just wondering, why not use webpack app/javascripts for new code?

Reply:
Yep, doing that right now. That was just simplest demo version using existing infrastructure.